bin/sf-format.js

```diff
@@ -9,4 +9,5 @@
 const shell = require('../utils/shelljs');
 
 // Simple one line command. If it needs to be customized, override script in sfdevrc file.
-shell.exec('prettier --write "+(src|test)/**/*.+(ts|js|json)"');
+shell.exec('prettier --write "+(src|test|schemas)/**/*.+(ts|js|json)"');
+shell.exec('prettier --write "command-snapshot.json"');
```

Comment:
This will return exit code 1 if a repo using dev-scripts doesn't have a snapshot file (core, sdr, stl):

![Screen Shot 2022-06-29 at 17 00 33](https://user-images.githubusercontent.com/6853656/176535993-0912cce2-614d-4b36-bc8f-6f290543c0a5.png)

Adding the snapshot file name to the current glob seems to work for all cases:
```ts
shell.exec('+(src|test|schemas)/**/*.+(ts|js|json)|command-snapshot.json');
```

Reply:
Ahhhh, thank you! I couldn't figure out how to get that to work correctly.